src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, correlation_id=None, application=None):
         self.extension_name = None
         self.extension_version = None
         self.event_id = str(uuid.uuid4())
+        self.recommendation_properties = None
```

Comment:
Are we adding the suffix `_properties` to indicate that it is a JSON object that contains many properties?

Reply:
I wanted to indicate by `_properties` that this is the property that contains all the recommendation information, but it seems that `recommendation_feedback` is a better name.

Reply:
How about `cli_recommendation_adoption`?

Reply:
Received, already modified in https://github.com/Azure/azure-cli/pull/25875/commits/176e59409fe89adaf19304fef5afb7b7f2ae3bc1.